### ISSUE TITLE
Offer bounties for completing the 1st trade in as-yet untraded fiat currencies

### ISSUE THREAD

**Issue by cbeams** [bisq-network/growth#34]
Goal is to see if we can bootstrap trading activity in as-yet untraded fiat currencies.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/42854234-26a819e0-8a3b-11e8-9943-e1dfd6a50958.png)

Somebody did the first Thai Baht trade! 

THB/BTC

**Comment by cbeams** [comment]
Good catch, @flix1. I just tweeted about it at https://twitter.com/bisq_network/status/1019449782207688704.
